### ISSUE TITLE
presentation-role-conflict should also fail on img with empty alt and aria-label

### ISSUE THREAD

**Issue by WilcoFiers** [dequelabs/axe-core#3702]
Axe-core should fail the following in the `presentation-role-conflict` rule:

```html
<img alt="" aria-label="Hello world" />
```

This image gets ignored in voiceover + Safari. This issue came up as part of ACT: https://wai-wcag-act-rules.netlify.app/standards-guidelines/act/implementations/axe-core/#breakdown-46ca7f

**Comment by padmavemulapati** [comment]
Validated with the latest develop branch code base, the test snippet:
`<img alt="" aria-label="Hello world" />` is failing for `presentation-role-conflict` rule



![Image](https://user-images.githubusercontent.com/47409579/196720186-3ef6bd15-cd3e-4132-9dfe-3f24f932205c.png)
